makefile.shade

```diff
@@ -203,6 +204,40 @@ var buildTarget = "compile"
         }
     }
 
+#cache-coherence
+    @{
+        var coherenceCacheDir = Path.Combine(BASE_DIR, Environment.GetEnvironmentVariable("COHERENCE_CACHE_DIR") ?? ".coherence-cache");
+        var dropsShare = Environment.GetEnvironmentVariable("ASPNETCI_DROPS_SHARE");
```

Comment:
There should be a default for this in code. It's hard to run it locally otherwise because you need to search through code to figure out what vars to set